ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/route-maps.j2

```diff
@@ -7,17 +7,17 @@
 {%     for route_map in route_maps | arista.avd.convert_dicts('name') | arista.avd.natural_sort('name') %}
 #### {{ route_map.name }}
 
-| Sequence | Type | Match and/or Set |
-| -------- | ---- | ---------------- |
+| Sequence | Type | Match and/or Set | Sub-Route-Map | Continue |
```

Comment:
Are the extra two columns necessary?  There's 150 files changed in this PR, virtually all due to this, with the vast majority of them these two columns just containing a '-'.  The "Match and/or Set" column could become a "Match/Action" column, and you could have continue or sub-route-map actions, with just another couple of if blocks as seen below.

Reply:
Oops, also didn't realise this was still a draft - apologies!

Reply:
Hi, i guess that could work but updating the column header would still cause the 150 files to change :-) I think it's more clear this way and i do think it's important to document the new knobs in the table.

Reply:
If we add them in separate columns, we might as well split Match and Set into their own columns as well.

Reply:
I think they should be separate as well. Should i add that in this PR though or separate one?

Reply:
this one